CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Minor: Improved error messaging of the `/clip` command. (#5879)
 - Minor: Added Linux support for Live Notifications toasts. (#5881)
 - Minor: Messages can now be deleted from the context menu in a channel. (#5956)
+- Minor: Show user avatar in Live Nofications on Linux. (#5971)
```

Comment:
This should be added to the original changelog entry (as we didn't have a release with notifications on Linux yet).